autopilot/agent_test.go

```diff
@@ -604,6 +604,7 @@ func TestAgentImmediateAttach(t *testing.T) {
 		},
 		Graph:           memGraph,
 		MaxPendingOpens: 10,
+		ConfTarget:      6,
```

Comment:
@Roasbeef @halseth do you think this functionality needs [unit] tested? 

If so, which test file(s) might I put them in? `agent_test.go` doesn't seem to make any assertions about fee rates.